downstream/modules/platform/proc-gw-searching-authenticator.adoc

```diff
@@ -0,0 +1,14 @@
+:_mod-docs-content-type: PROCEDURE
+
+[id="gw-searching-authenticator"]
+
+= Searching for an authenticator
+
+You can search for a previously configured authenticator from the Authentication list view.
+
+.Procedure
+
+. From the navigation panel, select {MenuAMAuthentication}. 
+. In the search bar, enter an appropriate keyword for the authentication method  you want to search for and click the arrow icon.
```

Comment:
If it's a > icon, there's one in the images directory

Reply:
It's actually a right arrow and not a > symbol so I'm going to leave it as is for now.